spec/system/admin_search_spec.rb

```diff
@@ -75,4 +76,12 @@ def open_search_modal
       "We couldn’t find anything matching ‘very long search phrase’.",
     )
   end
+
+  it "opens search modal when search input is clicked" do
+    visit "/admin"
+    sidebar.click_search_input
```

Comment:
I think clicking the search input should be the default action taken by the `open_search_modal` method so we can drop this test completely. Another test can be added for opening the modal with a keyboard shortcut.